### PR TITLE
Design

### PR DESCRIPTION
**Pull request by Hadeel-Salamin**
created the basic html file
created the style file
added image for background

### CODE REVIEW

**Review by amanshawar — APPROVED** [review]

**Review by Samaamro20 — APPROVED** [review]
well Done :+1:

File: public/style.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+label {
+	display: flex;
+	font-size: 19px;
+}
+
+form {
+	display: flex;
+	margin-left: 3%;
+	align-items: center;
+	justify-content: space-between;
```
Comment (m4v15) [inline public/style.css]: I think `space-around` looks a bit nicer 😅

File: public/style.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	margin-left: 6%;
+	margin-bottom: 10px;
+	color: #8F641B
+}
+
+#searchContainer {
+	display: flex;
+	flex-flow: row nowrap;
+	align-items: center;
+	margin-left: 3%;
```
Comment (m4v15) [inline public/style.css:24]: I would add `justify-content: space-around;`, it looks a lot nicer

File: public/style.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
+header {
+	height: 300px;
+	background-image: Url('music/15r.jpg');
```
Comment (m4v15) [inline public/style.css]: change this line so the image works or move the image to a music folder

**Review by m4v15 — CHANGES_REQUESTED** [review]

**Review by m4v15 — APPROVED** [review]

**Pull request merged**